src/universal/components/CreateCard/CreateCard.js

```diff
@@ -80,18 +69,41 @@ CreateCard.propTypes = {
   styles: PropTypes.object
 };
 
-const labelStyles = {
-  display: 'inline-block',
+const borderRadius = '.1875rem';
```

Comment:
I think most modern browser just default to zoom now, so it really doesn’t matter—they will scale up from `px` or `rem`...